Comp2024/src/main/java/frc/robot/commands/AutoStop.java

```diff
@@ -0,0 +1,59 @@
+
+// ROBOTBUILDER TYPE: Command.
+
+package frc.robot.commands;
+
+import com.ctre.phoenix6.mechanisms.swerve.SwerveRequest;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.CommandSwerveDrivetrain;
+
+/**
+ *
+ */
+public class AutoStop extends Command
+{
+  private final CommandSwerveDrivetrain          m_drivetrain;
+  private final SwerveRequest.ApplyChassisSpeeds request = new SwerveRequest.ApplyChassisSpeeds( );
+
+  public AutoStop(CommandSwerveDrivetrain drivetrain)
+  {
+    m_drivetrain = drivetrain;
+
+    setName("AutoStop");
+    addRequirements(m_drivetrain);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize( )
+  {
+
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute( )
+  {
+    m_drivetrain.setControl(request.withSpeeds(new ChassisSpeeds(0.0, 0.0, 0.0)));

```

Comment:
There might be a "request" that has the robot stopped (brake mode?) rather than create a new speed definition. Check if it exists.